lua/modules/plugins/lang.lua

```diff
@@ -8,11 +8,16 @@ lang["kevinhwang91/nvim-bqf"] = {
 		{ "junegunn/fzf", build = ":call fzf#install()" },
 	},
 }
-lang["fatih/vim-go"] = {
+lang["ray-x/go.nvim"] = {
 	lazy = true,
-	ft = "go",
-	build = ":GoInstallBinaries",
-	init = require("lang.vim-go"),
+	dependencies = { -- optional packages
+		"ray-x/guihua.lua",
+		"neovim/nvim-lspconfig",
+		"nvim-treesitter/nvim-treesitter",
+	},
+	ft = { "go", "gomod" },
+	build = ':lua require("go.install").update_all_sync()', -- if you need to install/update all binaries
```

Comment:
```suggestion
	build = ":GoInstallBinaries",
```

I think [`:GoInstallBinaries`](https://github.com/ray-x/go.nvim?tab=readme-ov-file#go-binaries-install-and-update) should be sufficient in this case since all the other packages are managed by Mason. What's the purpose of calling this function here?